cypress/integration/footer-feature.spec.js

```diff
@@ -0,0 +1,236 @@
+        toTopButton: 'scroll-to-top',
+        translateMenuItems: '.goog-te-menu2-item',
+        htmlTranslated: '.translated-ltr'
+    }
+}
+
+describe('Footer Features', () => {
+    
+    before(() => {
+        cy.visit(constants.rootUrl);
+    });
+
+    describe('Social Links', () => {
+        
+        it('should contain a facbook icon that points to the baltimore county facebook page', () => {
+            cy
+                .get(`${constants.selectors.socialLinks} .fa-facebook`)
+                .should('exist')
+                .parent()
+                .should('have.attr', 'href', 'https://www.facebook.com/bcplonline');
+        });
+
+        it('should contain a twitter icon that points to the baltimore county twitter page', () => {
+            cy
+                .get(`${constants.selectors.socialLinks} .fa-twitter`)
+                .should('exist')
+                .parent()
+                .should('have.attr', 'href', 'https://www.twitter.com/bcplinfo');
+        });
+
+        it('should contain a youtube icon that points to the baltimore county youtube page', () => {
+            cy
+                .get(`${constants.selectors.socialLinks} .fa-youtube`)
+                .should('exist')
+                .parent()
+                .should('have.attr', 'href', 'https://www.youtube.com/user/bcplonline');
+        });
+
+        it('should contain a flickr icon that points to the baltimore county flickr page', () => {
+            cy
+                .get(`${constants.selectors.socialLinks} .fa-flickr`)
+                .should('exist')
+                .parent()
+                .should('have.attr', 'href', 'https://www.flickr.com/photos/bcplphoto');
+        });
+
+        it('should contain a instagram icon that points to the baltimore county instagram page', () => {
+            cy
+                .get(`${constants.selectors.socialLinks} .fa-instagram`)
+                .should('exist')
+                .parent()
+                .should('have.attr', 'href', 'https://www.instagram.com/bcplinfo');
+        });
+
+    });
+
+    describe('Promotions', () => {
+        
+        it('should contain a link that navigates to the BCPL Home Page', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs} .logo`)
+                .should('have.attr', 'href', '/');
+        });
+
+        it('should contain a link that navigates to Admin Office address on Google Maps', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Directions')
+                .should('have.attr', 'href', "https://www.google.com/maps/dir/''/320+York+Rd,+Towson,+MD+21204");
+        });
+
+        it('should contain a link that navigates to the Ask A Librarian page', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Ask a Librarian')
+                .should('have.attr', 'href', "/services/ask-a-librarian.html");
```

Comment:
I think the href's that go to bcpl pages (relative links) you can go ahead and make sure they visit that page. This comment applies to the next two tests.